packages/peregrine/lib/talons/FilterModal/helpers.js

```diff
@@ -45,7 +48,17 @@ export const getStateFromSearch = (initialValue, filterKeys, filterItems) => {
 
             // map item values to items
             for (const value of params.getAll(key)) {
-                items.add(groupItemsByValue.get(value));
+                const existingFilter = groupItemsByValue.get(
+                    value.split(DELIMITER)[1]
```

Comment:
Out of curiosity, didn't you define a helper for this later in the file?

Reply:
Nice observation! I'll rename and re-use.